safe_extras/parameters/qt_widgets/test/test_boolean_parameter_widget.py

```diff
@@ -1,53 +0,0 @@
-# coding=utf-8
```

Comment:
Hi Ismail, I haven't had a detailed look at the surrounding code, but what is the rationale for removing these tests and what is the effect on test coverage?